bindinfo/bind_test.go

```diff
@@ -1688,3 +1705,71 @@ func (s *testSuite) TestIssue19836(c *C) {
 	)
 	tk.MustQuery("explain for connection " + strconv.FormatUint(tk.Se.ShowProcess().ID, 10)).Check(explainResult)
 }
+
+func (s *testSuite) TestIssue20417(c *C) {
```

Comment:
Could you please add this case https://github.com/pingcap/tidb/issues/20417#issuecomment-713279869 into this test to make sure the bind can be hit after this PR?

Reply:
There is already an equivalent test here.